internal/graph/dispatch_throttling_check_resolver.go

```diff
@@ -103,37 +84,20 @@ func (r *DispatchThrottlingCheckResolver) ResolveCheck(ctx context.Context,
 
 	currentNumDispatch := req.GetRequestMetadata().DispatchCounter.Load()
 
-	threshold := r.config.DefaultThreshold
-
-	maxThreshold := r.config.MaxThreshold
-	if maxThreshold == 0 {
-		maxThreshold = r.config.DefaultThreshold
-	}
-
-	thresholdInCtx := telemetry.DispatchThrottlingThresholdFromContext(ctx)
+	shouldThrottle := threshold.ShouldThrottle(
```

Comment:
nitpik: better to change shouldThrottle -> isThrottle to match the span attribute below

Reply:
I think `ShouldThrottle` gives better idea on it decides whether it should be throttled. for attributes `IsThrottled` would make sense but not for this method in my opinion.